### PR TITLE
monerod: remove outdated tor seed info

### PR DESCRIPTION
**Pull request by nahuhh**
Still needs some updates, but this is mostly to remove outdated tor seed recommendations

requires #46

https://36.md.monerodevs.org

### CODE REVIEW

File: docs/en/running-node/open-node-tor-onion.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ Feel free to adjust above to your preferred conventions, just remember to adjust
 
 [Download](/interacting/download-monero-binaries/) and [verify](/interacting/verify-monero-binaries/) the file.
 
-Extract `tar -xf monero-linux-x64-v0.17.1.9.tar.bz2` (adjust filename).
+Extract `tar -xf monero-*.tar.bz2` (adjust filename if necessary).
 
-Move binaries to `/opt/monero/` with `mv monero-x86_64-linux-gnu-v0.17.1.9/* /opt/monero/` then `chown -R monero:monero /opt/monero`
+Move binaries to `/opt/monero/` with `sudo mv monero-*/* /opt/monero/` then `sudo chown -R monero:monero /opt/monero`
```
Comment (plowsof) [inline docs/en/running-node/open-node-tor-onion.md:88]: have to be careful here and consider other possible folders in the dir monero-docs/* monero-gui/* monero-lws/* monero-python/* monero-site/*

Comment (plowsof) [reply docs/en/running-node/open-node-tor-onion.md:88]: would you like to use a {{ macro }} here for current monero version instead?

File: docs/en/running-node/open-node-tor-onion.md [hunk 2]
```diff
@@ -256,7 +234,7 @@ Proto Recv-Q Send-Q Local Address           Foreign Address         State
 ...
 tcp        0      0 0.0.0.0:18080           0.0.0.0:*               LISTEN      259255/monerod
 tcp        0      0 0.0.0.0:18081           0.0.0.0:*               LISTEN      259255/monerod
-tcp        0      0 127.0.0.1:18083         0.0.0.0:*               LISTEN      259255/monerod
+tcp        0      0 127.0.0.1:18084         0.0.0.0:*               LISTEN      259255/monerod
```
Comment (plowsof) [inline docs/en/running-node/open-node-tor-onion.md:237]: what's wrong with the default port of 18083 again?  slot of zmq operators have moved to away from the default port to 18084 https://www.reddit.com/r/Monero/comments/vga773/psa_to_public_node_operators_that_have_enabled/

Comment (nahuhh) [reply docs/en/running-node/open-node-tor-onion.md:237]: Default ports:
18080 p2p
18081 rpc
18082 zmq-rpc

User added:
18083 zmq-pub
18084 tor p2p
18085 i2p p2p
18089 restricted rpc

Keeping zmq's together and torp2p+i2pp2p together is cleaner than going zmq > tor > zmq > i2p

File: docs/en/running-node/open-node-tor-onion.md [hunk 3]
```diff
@@ -29,23 +29,23 @@ in a secure and private way over Tor. Your node will be always ready w/o delays
...
 
 **Tor onion for P2P network** is useful for other full node users as it allows them to broadcast transactions over Tor (using `--tx-proxy` option).
 
 ## Assumptions
 
-You understand basic Linux administration. You seek Monero specific guidance.
+You understand basic Linux administration.
 
-You have root access to a Linux server with 2GB+ RAM and 120GB+ SSD (or 50GB+ for the pruned node version). This is current for Jan 2021.
+You have root access to a Linux server with 4GB+ RAM and available SSD storage of 512GB+ or 256GB+ (preferred) for the full or pruned node. As of 2024-09-01, the current blockchain size is ~220GB and ~85GB, respectively.
```
Comment (plowsof) [inline docs/en/running-node/open-node-tor-onion.md:42]: food for thought: we could have a {{ macro }} for the current size on disk for a full / pruned node.. inside a 'note', e.g. 

Current blockchain sizes:
Fullnode size on disk: {{ lmdb_size_full }}
Pruned : {{ lmdb_size_pruned }}
Last updated : {{ lmdb_size_updated }}

Comment (DiosDelRayo) [reply docs/en/running-node/open-node-tor-onion.md:42]: This is the top idea! Great!  :100: 

About the text: I  think there is a small mistake and 256+ and 512+(prefered) are in reverse order... or is preferred related to the pruned node?

Comment (nahuhh) [reply docs/en/running-node/open-node-tor-onion.md:42]: Related to the full and pruned 👍👍 


```
“Respectively” is an adverb meaning “in the order mentioned.” We use it when we want to emphasize the relationships between two pairs or items in two parallel lists (i.e., two lists that contain the same number of items). For example, we could use “respectively” as follows:

Paris and Berlin are the capital cities of France and Germany, respectively. 

```

**Comment by plowsof** [comment]
https://github.com/monero-project/monero-docs/blob/f36a828799ace5a0762638cfb54e928ce8a4d347/docs/en/interacting/monerod-reference.md?plain=1#L143 

tx-proxy "requires" add-peer

**Comment by nahuhh** [comment]
> https://github.com/monero-project/monero-docs/blob/f36a828799ace5a0762638cfb54e928ce8a4d347/docs/en/interacting/monerod-reference.md?plain=1#L143 
> 
> tx-proxy "requires" add-peer. 

a) that doc is outdated.
tx-proxy does _not_ require add-peer. Monerod _does_ have onion and i2p seed nodes now


> mentions tx proxy being  "not recommended" to disable_noise and broadcast using anon network

b1) this was due to the poor connectivity (and b2) bugs when using i2p+tor) causing tx broadcasts to fail.

(A) was fixes before the bard fork
(B1) was fixed before the hardfork
(B2) was fixed more recenly recently

**Review by plowsof — APPROVED** [review]

**Pull request merged**